### ISSUE TITLE
Crash when drag and drop items between multiple notebooks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#322]
```
What steps will reproduce the problem?
1. Open notebook
2. Open another notebook, either as a tab or as a new instance of the program
3. Drag document from notebook A treeview into notebook B treeview

What is the expected output? What do you see instead?
Application crash just moments later. The treeview in the notebook from which I 
have dragged an item is gobbled (hard to describe, but items are missing, 
showing upon mouseover et c).

What version of the product are you using? On what operating system?
0.6.4 on Windows 7

Please provide any additional information below.
It seems to me that the drag and drop operation does indeed work, when I reload 
the notebooks and check.

```

Original issue reported on code.google.com by `shls...@gmail.com` on 20 Aug 2010 at 3:01
- Merged into: #25

**Comment by GoogleCodeExporter** [comment]
```
Note: What I meant by writing that the drag and drop works, is that it 
apparently succeeds. The operation was apparently executed when I re-open both 
notebooks and compare.
```

Original comment by `shls...@gmail.com` on 23 Aug 2010 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Moving notes between notebooks is not supported at this time.  There are 
several consistency checks that need to be in place before this can be done 
safely (ensure no duplicate nodeids, example).  I'm surprised the drag and 
dropped worked at all.  I tried to block that actually ;).  I am close to 
supporting moving between notebooks though, and when this is done, this bug 
should go away.  Thanks for the report.
```

Original comment by `matt.rasmus@gmail.com` on 10 Sep 2010 at 8:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 9 Dec 2010 at 3:56
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Same for me, it crashed but the drag&drop did work !

I may add that trying to drag a note from a subfolder to an upper-level folder 
didn't make KeepNote crash, but instead created an undeletable double of the 
said subfolder, along with a double of the trash bin (undeletable too, because 
of it being the trash bin itself).
```

Original comment by `hadrien....@gmail.com` on 29 Jan 2011 at 8:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 23 Feb 2011 at 3:59
- Changed state: **Duplicate**
